docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -65,6 +65,18 @@ def downstream_asset(upstream_asset):
 
 The [explicit dependencies](#explicit-dependencies) example covers an alternative way to specify asset dependencies without needing to match argument names to upstream asset names.
 
+### Using context in assets
```

Comment:
This is very important to include, but I think it would make more sense inside the examples section below. That will allow the page to cover the end-to-end set of asset-related concepts (asset definitions, dependencies groups, materializing), before going into the different options within those concepts.